solutions/instances/variables.tf

```diff
@@ -233,6 +246,21 @@ variable "enable_metrics_routing_to_cloud_monitoring" {
   default     = true
 }
 
+variable "metrics_router_settings" {
+  type = object({
+    default_targets = optional(list(object({
+      id = string
+    })), [])
+    permitted_target_regions  = optional(list(string))
+    primary_metadata_region   = optional(string, "eu-de")
+    backup_metadata_region    = optional(string, "us-east")
+    private_api_endpoint_only = optional(bool, false)
```

Comment:
default to private

Reply:
hmm, maybe not. It seems to cause an issue with UI:
![image](https://github.com/user-attachments/assets/95a5170b-08c5-4658-9360-cf02451aa4fb)